jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/UpdateChecker.java

```diff
@@ -121,12 +121,22 @@
         if (currentVersion.equals(version.latest)) {
           return Optional.empty();
         }
-        return Optional.of(
-            "A new version of Jib ("
+        String updateMessage =
+            "A new version of "
+                + toolName
+                + " ("
                 + version.latest
                 + ") is available (currently using "
                 + currentVersion
-                + "). Update your build configuration to use the latest features and fixes!");
+                + "). ";
+        if (toolName.equals("jib-cli")) {
```

Comment:
Hmm... although it works, I think this is too hacky to accept. Maybe we can make this method return only the latest version string (i.e., `version.latest`), and have the caller construct the message (maybe in `finishUpdateChecker`)? Or, any other thoughts?

Reply:
You're absolutely right. I refactored the code to make this method return `version.latest` and constructed the message in `finishUpdateCheck`.